src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/args/Args.java

```diff
@@ -126,13 +125,27 @@
    * @return [key1=value1,key2=value2,...], [] if keyValueMapping=empty/null
    */
   public static List<String> keyValues(Map<?, ?> keyValueMapping) {
+    return namedKeyValues("", keyValueMapping);
+  }
+
+  /**
+   * Produces a named key/value pair list given a name and a {@link Map}.
+   *
+   * @return [--name, key1=value1, --name, key2=value2,...], [] if keyValueMapping=empty/null
+   */
+  public static List<String> namedKeyValues(String name, Map<?, ?> keyValueMapping) {
     List<String> result = Lists.newArrayList();
```

Comment:
aren't we stuck with java7 though for this lib? even if so we can use guava to do stuff like this, although it will be much uglier.